packages/ActionBar/src/components/SearchInput/SearchInput.js

```diff
@@ -6,18 +6,23 @@ import SearchIcon from "@paprika/icon/lib/Search";
 import useI18n from "@paprika/l10n/lib/useI18n";
 import * as sc from "./SearchInput.styles";
 
-export default function SearchInput(props) {
-  const { a11yText, debouncedValue, onChange, placeholder, ...moreProps } = props;
+function useDebouncer(debounceDelay) {
```

Comment:
perhaps this could be a helper function on our `paprika/helpers` package as perhaps may not be the only place we will need this approach

Reply:
you're right